package.json

```diff
@@ -2078,7 +2078,6 @@
     "enabledApiProposals": [
         "notebookControllerKind",
         "notebookDebugOptions",
-        "notebookDocumentEvents",
```

Comment:
We don't need this anymore as it's published now. It was creating a compile error for me.